scalyr_agent/json_lib/parser.py

```diff
@@ -638,11 +645,14 @@ def __match(self, chars, error_message):
         and consume them.  In case of a mismatch, raise an exception.
         Only supports low-ASCII characters.
 
-        If error_message is None, we generate a default message."""
+        If error_message is None, we generate a default message.
```

Comment:
Is it still the case this only supports low-ASCII characters?

Reply:
Oh, right, it is not. Thanks.